test/widget_tests/splash_screen_test.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:trovami/Strings.dart';
+import 'package:trovami/helpers/RoutesHelper.dart';
+import 'package:trovami/ui/splash/splash_screen.dart';
+
+void main() {
```

Comment:
I haven't used tests like this. You'll have to teach it to me.  Does it run as a separate app?

Reply:
No, it's the same app. No different tool required. It's part of the flutter sdk. The one there is a widget test, https://flutter.dev/docs/cookbook/testing/widget/introduction

Reply:
You basically write the test and run `flutter test`